handlers/main.yml

```diff
@@ -1,3 +1,7 @@
 ---
 - name: Restart marathon
   service: name=marathon state=restarted
+
+- name: Reload daemon
+  command: systemctl daemon-reload
```

Comment:
tja, If we could have a statistic about the current ansible-versions used !
IMO it's a bit early to require 2.2.x, but requiring >2(.1).x OK for me.
To be honest in our company <30% upgraded to 2.2.